doc/rst/platforms/comm-layers/libfabric.rst

```diff
@@ -318,6 +318,25 @@ across the nodes assigned to the job.  Overloading can still be an issue
 if there are more Chapel locales (program instances) than nodes in the
 slurm job, however.
 
+Other Settings
+--------------
+
+Setting these two environment variables can improve performance.
+
+.. code-block:: bash
+
+   export CHPL_RT_COMM_OFI_DEDICATED_AMH_CORES=true
```

Comment:
Also, can you add a link from the aws docs (which mention these variables) back to this new section?